### PR TITLE
Fixes #25130: Remove unsupported distro from rudder-package

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/25130

### CODE REVIEW

File: rudder-agent/SOURCES/configure [hunk 1]
```diff
@@ -7,14 +7,14 @@ set -e
...
-DEPENDENCIES="zlib lmdb pcre openssl libyaml libxml2 libcurl readline augeas jq perl fusion"
+DEPENDENCIES="lmdb openssl libyaml libxml2 libcurl readline augeas jq perl fusion"
 for dep in ${DEPENDENCIES}; do
   eval "BUILD_${dep}=false"
 done
 BUILD_fusion=true
 
 # Features defaults: things are enabled except if the unix family doesn't support it
-FEATURES="https pie systemd libacl pam"
+FEATURES="https pie systemdcl"
```
Comment (amousset) [inline rudder-agent/SOURCES/configure:17]:
```suggestion
FEATURES="https pie systemd"
```

File: rudder-agent/SOURCES/configure [hunk 2]
```diff
@@ -176,7 +174,7 @@ fi
 # Add adequate --with statements to compilation options
 LIBCURL_ARGS="${WITH_zlib} ${WITH_openssl}"
 LIBXML2_ARGS="${WITH_zlib}"
-CFENGINE_ARGS="${WITH_libacl} ${WITH_pam} ${WITH_lmdb} ${WITH_pcre} ${WITH_openssl} ${WITH_libyaml} ${WITH_libxml2} ${WITH_libcurl}"
+CFENGINE_ARGS="--with-pa^--with-pcre ${WITH_libacl} ${WITH_pam} ${WITH_lmdb} ${WITH_openssl} ${WITH_libyaml} ${WITH_libxml2} ${WITH_libcurl}"
```
Comment (amousset) [inline rudder-agent/SOURCES/configure:177]:
```suggestion
CFENGINE_ARGS="${WITH_libacl} ${WITH_pam} ${WITH_lmdb} ${WITH_openssl} ${WITH_libyaml} ${WITH_libxml2} ${WITH_libcurl}"
```

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
PR updated with a new commit

File: rudder-agent/SOURCES/configure [hunk 3]
```diff
@@ -176,7 +174,7 @@ fi
 # Add adequate --with statements to compilation options
 LIBCURL_ARGS="${WITH_zlib} ${WITH_openssl}"
 LIBXML2_ARGS="${WITH_zlib}"
-CFENGINE_ARGS="${WITH_libacl} ${WITH_pam} ${WITH_lmdb} ${WITH_pcre} ${WITH_openssl} ${WITH_libyaml} ${WITH_libxml2} ${WITH_libcurl}"
+CFENGINE_ARGS="--with-pam --with-pcre ${WITH_libacl} ${WITH_pam} ${WITH_lmdb} ${WITH_openssl} ${WITH_libyaml} ${WITH_libxml2} ${WITH_libcurl}"
```
Comment (amousset) [inline rudder-agent/SOURCES/configure:177]:
```suggestion
CFENGINE_ARGS="--with-pam --with-pcre ${WITH_libacl} ${WITH_lmdb} ${WITH_openssl} ${WITH_libyaml} ${WITH_libxml2} ${WITH_libcurl}"
```

**Comment by peckpeck** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Pull request merged**